docker-compose-agent-ipfs.yml

```diff
@@ -28,9 +27,6 @@ services:
     depends_on:
       ipfs0:
         condition: service_healthy
-    environment:
-      # possible to set values using env variables
-      AFJ_REST_LOG_LEVEL: 1
```

Comment:
as far as I can tell this env does nothing and log level is set in config files e.g. `samples/cliConfig.json`